fab/fabfile.py

```diff
@@ -551,6 +519,14 @@ def record_successful_deploy(url):
 
 
 @task
+@roles(ROLES_ALL_SRC)
+@parallel
+def record_successful_release():
+    with cd(env.root):
+        files.append(RELEASE_RECORD, str(env.code_root), use_sudo=True)
```

Comment:
can do, though this is mainly here to keep track of which releases we can delete in the `releases` directory. if a release isn't recorded, i assume it was a preindex or a failed deploy and wipe it